engine/src/main/java/org/destinationsol/ui/UiDrawer.java

```diff
@@ -87,7 +88,7 @@ public void drawString(String s, float x, float y, float scale, boolean centered
     }
 
     public void drawString(String s, float x, float y, float scale, TextAlignment align, boolean centered, Color tint) {
-        drawer.drawString(s, x, y, scale * FONT_SIZE, align, centered, tint);
+        drawer.drawString(s, x, y, scale * FONT_SIZE * FONT_SIZE_MULTIPLIER / displayDimensions.getHeight(), align, centered, tint);
```

Comment:
We are *definitely* not computing the true value of fontSize on every call to drawString.

Remove FONT_SIZE and FONT_SIZE_MULTIPLIER, and addd two new variables - `FONT_SIZE_IN_PX = 20` and `fontSize = someCalculation;.`